scripts/cdn_publish.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 cdnGravityDir="gravity-ui-web"
-cdnDir="cdn-dist"
+cdnDistDir="cdn-dist"
+cdnCurrentVersionDir="$cdnDistDir/currentVersion"
+cdnCurrentVersionGravityDir="$cdnCurrentVersionDir/$cdnGravityDir"
+cdnLatestVersionsDir="$cdnDistDir/latestsVersions"
```

Comment:
Minor typo: "latest**s**"

Also minor request, I think file/folder names in the gravity repo tend to be hyphen cased rather than camel cased. Might be nice to keep to that even though it would only be a cosmetic difference in this case: `current-version` and `latest-version`

Reply:
Doh!